app/controllers/users_controller.rb

```diff
@@ -1,16 +1,20 @@
 class UsersController < ApplicationController
-  before_action :set_user, only: [:show]
+  before_action :set_user, only: %i(edit show index)
```

Comment:
```suggestion
  before_action :set_user, only: %i(edit show)
```
index đâu cần load_user, lỗi action do k truyền lên params id